gatsby-node.ts

```diff
@@ -271,27 +269,22 @@ export const createPages: GatsbyNode<any, Context>["createPages"] = async ({
           const splitSlug = slug.split("/")
           splitSlug.splice(1, 1, lang)
           const langSlug = splitSlug.join("/")
-          createPage({
+          createPage<Context>({
             path: langSlug,
             component: path.resolve(`src/templates/${template}.tsx`),
             context: {
+              language: lang,
```

Comment:
Keeping most of these context values for backward compatibility. I'll probably refactor this in next iterations.